src/styles.css

```diff
@@ -176,6 +176,18 @@
   src: url("https://sunflower-land.com/testnet-assets/fonts/Secondary.ttf");
 }
 
+@font-face {
+  font-family: "Born2bSporty";
+  src: url("/fonts/Born2bSportyFS.otf");
+  /* src: url("https://sunflower-land.com/testnet-assets/fonts/Born2bSportyFS.otf"); */
+}
+
+@font-face {
+  font-family: "Basis33";
```

Comment:
basis33 is licensed under the following terms:
custom [LICENSE.txt](https://st.1001fonts.net/license/basis33/LICENSE.txt) within zip-file [basis33.zip](https://www.1001fonts.com/download/basis33.zip)
[1001Fonts general font usage terms](https://www.1001fonts.com/licenses/general-font-usage-terms.html)

Free for commercial use